lib/instrumentation/span.js

```diff
@@ -87,6 +88,11 @@ Span.prototype.setHttpContext = function (context) {
   this._http = Object.assign(this._http || {}, context)
 }
 
+Span.prototype.setDestinationContext = function (context) {
+  if (!context) return
```

Comment:
What's the reason for this guard? As far as I can see based on the current code, `context` should always be set, right?